api/api-rules/violation_exceptions.list

```diff
@@ -360,6 +360,7 @@ API rule violation: list_type_missing,k8s.io/kubelet/config/v1beta1,CredentialPr
 API rule violation: list_type_missing,k8s.io/kubelet/config/v1beta1,KubeletConfiguration,AllowedUnsafeSysctls
 API rule violation: list_type_missing,k8s.io/kubelet/config/v1beta1,KubeletConfiguration,ClusterDNS
 API rule violation: list_type_missing,k8s.io/kubelet/config/v1beta1,KubeletConfiguration,EnforceNodeAllocatable
+API rule violation: list_type_missing,k8s.io/kubelet/config/v1beta1,KubeletConfiguration,ProxiedEndpoints
```

Comment:
I used [this](https://github.com/kubernetes/kubernetes/pull/102915/commits/d82f6069708ff25ecff48464bee82bc570b55641#r740407436) comment to justify this change